cwms-data-api/src/test/resources/cwms/cda/data/dto/location/kind/gate-settings.json

```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "type": "gate-setting",
```

Comment:
this data structure needs to scale similar to time series

Reply:
Do you have some time today or this week where we can talk about this?